src/domains/ecommerce-app/api/npg-notification/_base_policy.xml.tpl

```diff
@@ -0,0 +1,28 @@
+<policies>
+
+  <inbound>
+      <cors>
+        <allowed-origins>
+          <origin>*</origin>
+        </allowed-origins>
+        <allowed-methods>
+          <method>POST</method>
+        </allowed-methods>
+      </cors>
+      <base />
+      <rate-limit-by-key calls="150" renewal-period="10" counter-key="@(context.Request.Headers.GetValueOrDefault("X-Forwarded-For"))" />
```

Comment:
to remove :-)

Reply:
Fixed with https://github.com/pagopa/pagopa-infra/pull/1263/commits/3aac28cbfb0d17e22065bf61954bc88c386af1c1